content/docs/instrumenting/exporters.md

```diff
@@ -148,6 +148,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [Confluence exporter](https://github.com/AndreyVMarkelov/prom-confluence-exporter)
    * [Dovecot exporter](https://github.com/kumina/dovecot_exporter)
    * [eBPF exporter](https://github.com/cloudflare/ebpf_exporter)
+   * [IBM Z HMC exporter](https://github.com/zhmcclient/zhmc-prometheus-exporter)
```

Comment:
This should probably go in the "Hardware related" section, where we have stuff like the IPMI exporter?